### PR TITLE
Use our own testsite in firefox html5 test

### PR DESCRIPTION
**Pull request by asdil12**
https://progress.opensuse.org/issues/110665

### CODE REVIEW

**Comment by asdil12** [comment]
https://openqa.suse.de/tests/9132731

File: tests/x11/firefox/firefox_html5.pm [hunk 1]
```diff
@@ -22,24 +21,8 @@ sub run {
...
-        }
-        elsif (match_has_tag('firefox-youtube-signin')) {
-            assert_and_click('firefox-youtube-signin');
-            wait_still_screen(2);
-            next;
-        }
-        last;
-    }
-    send_key_until_needlematch('firefox-testvideo', 'spc', 30, 5);
+    $self->firefox_open_url('os-autoinst.github.io/os-autoinst-distri-opensuse/testwebsites/html5_video');
```
Comment (mloviska) [inline tests/x11/firefox/firefox_html5.pm:24]: Is this URL valid ?

Comment (asdil12) [reply tests/x11/firefox/firefox_html5.pm:24]: It should be valid after this PR got merged and the GitHub action ran through.

**Comment by asdil12** [comment]
Note that before merging I need to fix the order of the test nodules to something that makes more sense.

File: tests/x11/firefox/firefox_html5.pm (hunk 1 above)
Comment (foursixnine) [reply tests/x11/firefox/firefox_html5.pm:24]: @asdil12 can you add a follow up ticket  to have that running in an opensuse.org resource?

Comment (asdil12) [reply tests/x11/firefox/firefox_html5.pm:24]: the python webserver idea is not so bad - I will give it a try

**Pull request merged**